### ISSUE TITLE
Integrating with Django's Sessions (make it use JWTs too!)

### ISSUE THREAD

**Issue by elnygren** [GetBlimp/django-rest-framework-jwt#270]
I found myself in a situation where I was using Django, DRF and DRF-JWT and wanted to make Django's regular session cookies be JWT tokens that I could use for API requests. I started with having a two-cookie setup but figured that maybe it would be handier to just make Django issue JWT-token-in-a-cookie cookies.

Anyway here's what I came up with: https://gist.github.com/elnygren/830e560e9a167464774693cbd55389c9

tl;dr a new SessionStore based on Django's signed_cookie SessionStore and a middleware that populates the session with whatever is needed by DRF-JWT (e.g the username field)

**Comment by megapctr** [comment]
Why not just use drf's SessionAuthentication for that?

**Comment by woosungchu** [comment]
In my case, SessionAuthentication is not working properly for jwt. It worths thinking about how to handle sesssion in jwt environment.

**Comment by godd9170** [comment]
Is it possible to use DRF SessionAuthentication while making API requests from a client whose domain is not the Django site? Say for example... a chrome extension? I'd like to be able to use an existing session cookie (which I can gain access to from the extension) as the Authorization for my API.

Thoughts?
